code/src/components/Start.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import TypeIt from 'typeit-react'
+import styled from 'styled-components/macro'
+import { maze } from 'reducers/maze'
+import { FetchGame } from './FetchGame'
+import { Loading } from './Loading'
+import { Button } from './Buttons'
+
+export const Start = () => {
+  const currentLocation = useSelector((store) => store.maze)
+  const [userName, setUserName] = useState('')
+  const dispatch = useDispatch()
+
+  const onNameSubmit = (event) => {
+    event.preventDefault()
+    dispatch(maze.actions.setUser(userName))
+    dispatch(FetchGame())
+  }
+  if (currentLocation.isLoading) return <Loading />
+  return (
+    <StartPage>
+      <TypeIt
+        options={{ speed: 50, cursor: false }}>
+        <StartText>
+          <p>
+            You have a burning ache in your throat
```

Comment:
Really nice that you have added your own beginning and ending, it adds some storytelling to the user's experience.